src/Npgsql/Replication/TestDecoding/NpgsqlTestDecodingPluginOptions.cs

```diff
@@ -0,0 +1,86 @@
+﻿using JetBrains.Annotations;
+using System.Collections.Generic;
+
+namespace Npgsql.Replication.TestDecoding
+{
+    /// <summary>
+    /// Options to be passed to the test_decoding plugin
+    /// </summary>
+    public readonly struct NpgsqlTestDecodingPluginOptions
```

Comment:
One thing I wanted to do (but forgot) is convert this to a class (or `NpgsqlPgOutputReplicationOptions` to a struct) for consistency reasons.
What do you think @roji

Reply:
Sounds good to me. Yeah, this is a somewhat fat struct, and there's absolutely no perf concern here whatsoever - class sounds more appropriate. But then we should also define equality properly etc.

Reply:
Done in 744e7b91f40155f439351043fb7ebbbdd1834e44

Reply:
Done in 744e7b91f40155f439351043fb7ebbbdd1834e44